From e84d279b900223724e7b81c97ec6b0bab30381a9 Mon Sep 17 00:00:00 2001
From: David Mitchell <davem@iabyn.com>
Date: Sun, 24 Feb 2013 15:45:48 +0000
Subject: fix off-by-two error in List::Util

A string literal is being used that includes two bytes beyond the
end of the string.

Origin: http://perl5.git.perl.org/perl.git/commit/623a911da450f8f4f1f400cb2c291c7898aecbd1
Patch-Name: fixes/list_util_off_by_two.diff
---
 cpan/List-Util/ListUtil.xs |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/List-Util/ListUtil.xs b/cpan/List-Util/ListUtil.xs
index 7da9b95..eacdde4 100644
--- a/cpan/List-Util/ListUtil.xs
+++ b/cpan/List-Util/ListUtil.xs
@@ -595,7 +595,7 @@ BOOT:
     varav = GvAVn(vargv);
 #endif
     if (SvTYPE(rmcgv) != SVt_PVGV)
-	gv_init(rmcgv, lu_stash, "List::Util", 12, TRUE);
+	gv_init(rmcgv, lu_stash, "List::Util", 10, TRUE);
     rmcsv = GvSVn(rmcgv);
 #ifndef SvWEAKREF
     av_push(varav, newSVpv("weaken",6));
